tests/README.md

```diff
@@ -2,11 +2,15 @@
 
 Examples are tested using tool [htrun](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests) and templated print log. 
 
-To run the test, use following command after you build the example:
+To run the test, use following command after you build the example: 
 ```
-mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\sys-info.bin --compare-log tests\sys-info.log
-```
-
-
-More details about `htrun` are [here](https://github.com/ARMmbed/htrun#testing-mbed-os-examples).
+$ mbedhtrun -d <MOUNT_POINT> -p <SERIAL_PORT> -m <TARGET> -f .\BUILD\<TARGET>\<TOOLCHAIN>\mbed-os-example-sys-info.bin --compare-log tests\sys-info.log
+``` 
 
+For example:
+```
+mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\mbed-os-example-sys-info.bin --compare-log tests\sys-info.log
+``` 
+More details about 
+- `htrun` are [here](https://github.com/ARMmbed/htrun#testing-mbed-os-examples).
```

Comment:
The link is obsolete. In any case a link is already provided at the top so it is not necessary to repeat.